src/geo/jts.clj

```diff
@@ -1,37 +1,138 @@
 (ns geo.jts
   "Wrapper for the locationtech JTS spatial library. Constructors for points,
   coordinate sequences, rings, polygons, multipolygons, and so on."
+  (:require [clojure.string :as str])
```

Comment:
Small issue but I'd rather not use this alias since it shadows the default `str` function -- if anything I'd just refer the ones you're using like `(:require [clojure.string :refer [join]])` or whatever